core/dbt/include/global_project/macros/strategies/materialized_view/full_refresh.sql

```diff
@@ -0,0 +1,12 @@
+{% macro strategy__materialized_view__full__refresh(relation, sql, backup_relation=None) %}
```

Comment:
There's an extra _ in `full__refresh` (also appears below and when called in the materialization).

Do we need to pass in `backup_relation`? I'm on the fence about it. But if we do, then we would need the intermediate relation as well, so that we can do the swap. I think the order would look like:
1. Create the new view as `intermediate_relation`
2. Swap names: `target_relation` > `backup_relation`
3. Swap names: `intermediate_relation` > `target_relation`
4. Drop `backup_relation`